### ISSUE TITLE
Failed to inject Dock.app SONOMA 14.6

### ISSUE THREAD

**Issue by encinascarles** [koekeishiya/yabai#2373]
I updated my mac to SONOMA 14.6 yesterday and I haven't been able to get the scripting addition to work.

When I run `sudo yabai --load-sa` I get `yabai: scripting-addition failed to inject payload into Dock.app!` and then I get a popup saying Dock.app crashed. The automatic scripting-adition load seems to be working because when I restart I also get the popup saying Dock.app crashed. I suppose it hasn't been updated yet to 14.6? I had been 6 months without updating just for yabai to work, but when I saw it was even half-working for Sequoia I assumed it must be working with all Sonoma versions, but I didn't know that actually Sonoma 14.6 came later...

I did the following:
- ✅ Installed the last version with brew
- ✅ Restarted multiple times
- ✅ Tried to install it manually without brew but didn't work either
- ✅ Tried `sudo nvram boot-args=-arm64e_preview_abi` as specified by [this](https://github.com/koekeishiya/yabai/issues/1158#issuecomment-1060168503)

**Comment by alwyn** [comment]
It happens on about every mac os update.  Best is to delay, if you can, updating until an updated version of yabai is released.

**Comment by koekeishiya** [comment]
yabai v7.1.1 works fine for me on Apple Silicon, and I also checked the Dock for x86-64 and it should also be compatible already. Follow the wiki install instructions and check that SIP didn't get reset for some reason.

**Comment by encinascarles** [comment]
Issue solved! Thank you for the help.

I tried re-enabling SIP, but it was already enabled. After further investigation, I discovered an old version of Yabai installed outside of Homebrew, located in a folder. I uninstalled it, and now everything works perfectly.

Thank you again. I couldn't stand living without Yabai these past few days. I primarily use it for quick space switching and moving windows between spaces, but it has become indispensable for me.
